tests/objectgraph_test.go

```diff
@@ -136,6 +152,101 @@ var _ = Describe("[sig-storage]ObjectGraph", decorators.SigStorage, func() {
 				Expect(child.Labels["kubevirt.io/dependency-type"]).To(Equal("storage"))
 			}
 		})
+
+		It("should detect hotplugged disks in the object graph", func() {
+			By("Hotplugging a disk")
+			hotplugPVC := libstorage.NewPVC("hotplug-pvc", "1Gi", "")
+			hotplugPVC, err := virtClient.CoreV1().PersistentVolumeClaims(testsuite.GetTestNamespace(nil)).Create(context.Background(), hotplugPVC, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			hotplugOptions := &v1.AddVolumeOptions{
+				Name: hotplugPVC.Name,
+				Disk: &v1.Disk{
+					DiskDevice: v1.DiskDevice{},
+					Serial:     hotplugPVC.Name,
+				},
+				VolumeSource: &v1.HotplugVolumeSource{
+					PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
+						PersistentVolumeClaimVolumeSource: corev1.PersistentVolumeClaimVolumeSource{
+							ClaimName: hotplugPVC.Name,
+						},
+					},
+				},
+				DryRun: nil,
+			}
+			Expect(virtClient.VirtualMachine(vm.Namespace).AddVolume(context.Background(), vm.Name, hotplugOptions)).ToNot(HaveOccurred())
+
+			By("Getting the object graph for the VM")
```

Comment:
Should this be wrapped in an Eventually, I am not sure if AddVolume waits for the hotplugged volume to be fully added before returning. If not it looks like a potential race between the hotplug completing and the graph pulling the information. Or don't we care if the hotplugged completed, just that it is there?